app/models/rook.rb

```diff
@@ -1,2 +1,6 @@
 class Rook < Piece
+  def piece_rules?(to_x:, to_y:)
+    return true if to_y == y_position || to_x == x_position
```

Comment:
Actually the way the &&'s and the || works after playing with it, this would actually work for the vertical movement check:
`  return true if to_y > y_position && to_x == x_position || to_y < y_position && to_x == x_position`